src/mock-builders/TranslationContext.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { PropsWithChildren } from 'react';
+import moment from 'moment';
+import { TranslationContextValue, TranslationProvider } from '../context';
+
+export const translationProviderData = { t: String, tDateTimeParser: moment };
+
+type TranslationProviderMockProps = PropsWithChildren<{ value?: TranslationContextValue }>;
+
+export const TranslationProviderMock = ({
```

Comment:
mocking this is nice but it's also possible to just use the `TranslationProvider` with a default value directly in the test

Reply:
Removing the mock altogether.